sql/util_time.sql

```diff
@@ -46,3 +52,25 @@ RETURNS INT8 AS '@MODULE_PATHNAME@', 'ts_continuous_agg_watermark_materialized'
 
 CREATE OR REPLACE FUNCTION _timescaledb_functions.subtract_integer_from_now( hypertable_relid REGCLASS, lag INT8 )
 RETURNS INT8 AS '@MODULE_PATHNAME@', 'ts_subtract_integer_from_now' LANGUAGE C STABLE STRICT;
+
+CREATE OR REPLACE FUNCTION _timescaledb_functions.as_multirange(low bigint, high bigint, tstzrange)
```

Comment:
it would be good to know that the input is apart from bigint. I assume this is our internal UNIX time? Maybe we can make that clear in the function or parameter names?

Or, could we create an alias type for UNIX time?

Reply:
I would prefer an concrete type for Unix time that support all functionality supported by PostgreSQL timestamps, but this is out of scope for this PR.

Added a comment that these are Unix timestamps and modified the names to contain "usec".